src/bisect.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+/* @flow */
+
+var sign = require('./sign');
+/**
+ * [Bisection method](https://en.wikipedia.org/wiki/Bisection_method) is a root-finding 
+ * method that repeatedly bisects an interval to find the root.
+ * 
+ * This function returns a numerical approximation to the exact value.
+ * 
+ * @param {Function} f input function
+ * @param {Number} a start of interval
+ * @param {Number} b end of interval
+ * @param {Number} nMax the maximum number of iterations
+ * @param {Number} tol the error tolerance
+ * @returns {Number} estimated root value
+ * 
```

Comment:
Since this method can throw an error, can you add a [throws tag](http://usejsdoc.org/tags-throws.html) to make that expectation clear to users?